src/app/authentication/repositories/validateOTPRepository.go

```diff
@@ -0,0 +1,58 @@
+package repositories
+
+import (
+	"github.com/go-sql-driver/mysql"
+	"gorm.io/gorm"
+	"log"
+	"stock_broker_application/src/constants"
+	"stock_broker_application/src/models"
+	"time"
+)
+
+type CustomerRepository interface {
```

Comment:
keep all the variable names as UserRepository as table name is user. customer is not related.